"error: suggest braces around initialization of subobject"

I'm currently getting this error while trying to build the compiler [1]. 
This might be due to some quirk of my system (macOS, just updated to 11.0.1). I'm not sure whether the fix is to add the suggested braces or to silence the warning. I found [this thread](https://bugs.llvm.org/show_bug.cgi?id=21629) on the llvm site, but I'm not sure if it's the same case that we have here.

[1]
```
> make
gcc -std=c99 -Wall -Wextra -Wno-unused-function -Wno-unused-parameter -Werror -pedantic -o bin/mirth0 bin/mirth0.c
bin/mirth0.c:74:47: error: suggest braces around initialization of subobject [-Werror,-Wmissing-braces]
static volatile value_t stack [STACK_SIZE] = {0};
                                              ^
                                              {}
bin/mirth0.c:80:44: error: suggest braces around initialization of subobject [-Werror,-Wmissing-braces]
static volatile cell_t heap [HEAP_SIZE] = {0};
                                           ^
                                           {}
2 errors generated.
make: *** [bin/mirth0] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-C99FLAGS=-std=c99 -Wall -Wextra -Wno-unused-function -Wno-unused-parameter -Werror -pedantic
+C99FLAGS=-std=c99 -Wall -Wextra -Wno-unused-function -Wno-unused-parameter -Wno-missing-braces -Werror -pedantic
 CC=gcc $(C99FLAGS)
 
 SRCS=src/*.mth src/prelude/*.mth src/mirth/*.mth src/mirth/data/*.mth src/mirth/elab/*.mth src/mirth/lexer/*.mth

